k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
+  labels:
+#The following label will be used for additional node reference, since we go for UID based selection of pod.
+    "kubernetes.io/hostname": "node-host-label"
+
+#finalizer helps handling deletion of resource
+  finalizers: ["cstorpool.openebs.io/finalizer"]
+
+spec:
+  disks:
+   diskList: ["/dev/sdc1"]
```

Comment:
do we really need to have a property called diskList

Reply:
As discussed, this diskList([ ] string) will be changed to disk([ ] struct) in another pull request.